### PR TITLE
Convert to cs proj

### PR DESCRIPTION
**Pull request by richardjharding**
These changes are  a result of migrating the existing project.json based projects to the new csproj form.
I've updated the Specflow.NetCore to infer the test framework in use by looking at the package references in the migrated csproj files - this is a little less robust than the more explicit test runner setting in the project json.

In order to test the changes I used a local nuget.config file to point to the local nupkg - I've not commited those files.

### CODE REVIEW

File: src/SpecFlow.NetCore/Fixer.cs [hunk 1]
```diff
@@ -65,9 +65,9 @@ public void Fix(DirectoryInfo directory)
 			_featureFiles = directory.GetFiles("*.feature", SearchOption.AllDirectories);
 			var missingGeneratedFiles = _featureFiles.Where(f => !File.Exists(f.FullName + ".cs")).ToList();
 
-			var xproj = GetXproj(directory);
+			var xproj = GetCsProj(directory);
```
Comment (erik-inkapool) [inline src/SpecFlow.NetCore/Fixer.cs]: Minor: this var is better named as `csproj`

**Review by erik-inkapool — COMMENTED** [review]
Awesome work! This looks really good to me!

File: src/SpecFlow.NetCore/Fixer.cs (hunk 1 above)
Comment (richardjharding) [reply src/SpecFlow.NetCore/Fixer.cs]: whoops knew I'd miss one!

**Comment by stajs** [comment]
Great job! I'll merge this in, but as I mentioned before I don't actually have the latest tooling yet (still waiting on some work stuff to settle down) so I won't be able to test it out and get it up on nuget.org until that's sorted. In the meantime if it's a blocker, you can chuck the nuget package up somewhere I can grab it, and I'll upload it to nuget.org on faith.

Can I also ask you to have a look at updating the README with relevant changes?

**Comment by richardjharding** [comment]
Thanks sure I'll do the README - not done many of these - is that a new branch now?

For the nuget - shall I leave version as RC7?

**Comment by stajs** [comment]
> Thanks sure I'll do the README - not done many of these - is that a new branch now?

Well branching is completely up to you, but I would suggest nah....

> For the nuget - shall I leave version as RC7?

Please bump to RC8. In fact, it would be good if you can include that bump with the PR for the README change as well.

**Pull request merged**